### PR TITLE
Add release notes for Hybrid App Base 4.2.0

### PR DESCRIPTION
**Pull request by smarki**

### CODE REVIEW

File: content/releasenotes/mobile/hybrid-app.md [hunk 1]
```diff
@@ -12,7 +12,19 @@ If you are using the advanced flow, you can get the latest version of the Hybrid
 
 For more information on hybrid mobile app development in Mendix, see the [Hybrid Mobile](/refguide/hybrid-mobile) section of the *Studio Pro 8 Guide* and the [Hybrid Mobile](/howto/mobile/hybrid-mobile) section of the *Studio Pro 8 How-to's*.
 
-## 2019
+## 2020
+
+### Hybrid App Base 5.0.0
+
+**Release date: April 15th, 2020**
```
Comment (ConnorLand) [inline content/releasenotes/mobile/hybrid-app.md:19]: Is this the date we are shooting for?

File: content/releasenotes/mobile/hybrid-app.md [hunk 2]
```diff
@@ -12,7 +12,19 @@ If you are using the advanced flow, you can get the latest version of the Hybrid
...
+
+With this release we updated all PhoneGap dependencies to the latest supported versions to comply with the Apple App Store guidelines for the deprecated UIWebView dependency:
```
Comment (ConnorLand) [inline content/releasenotes/mobile/hybrid-app.md:21]: The bulleted list below are the dependencies, right?

File: content/releasenotes/mobile/hybrid-app.md (hunk 1 above)
Comment (smarki) [reply content/releasenotes/mobile/hybrid-app.md:19]: yes

File: content/releasenotes/mobile/hybrid-app.md (hunk 2 above)
Comment (smarki) [reply content/releasenotes/mobile/hybrid-app.md:21]: Yes

**Pull request merged**